### ISSUE TITLE
snake game sprites

### ISSUE THREAD

**Issue by itspyguru** [CnL26/MyPixelArtChars#3]
Hey bro, where are the fruits assets in the snake game. They are in the preview but missing in the spritesheet.

**Comment by CnL26** [comment]
You have to check the pack from itch I'm about to update it in 1 hour!!!
Take the update you will like it more!

**Comment by itspyguru** [comment]
Sure.

**Comment by CnL26** [comment]
https://cnl26.itch.io/lorenzos-snake-game-assets

**Comment by CnL26** [comment]
You're going to have to experiment with the backgrounds

**Comment by CnL26** [comment]
Tell me what you think it does look a lot better without the borders I had before

**Comment by CnL26** [comment]
Did you see the updates?

**Comment by itspyguru** [comment]
I have started working on the game from yesterday. Here's the source code : https://github.com/pyGuru123/Python-Games/tree/master/Snake

The arcade mode works for now. I haven't implemented the snake skin yet as its little difficult. I'll tell you if i need any sprite. Also can you create a pixel art apple tree. You can check a sample tree in the assets/tree. I need something or same like this.

**Comment by CnL26** [comment]
Ok, I'll take a look.

**Comment by CnL26** [comment]
What size in pixels would you like the tree to be?

**Comment by itspyguru** [comment]
now all 3 modes work without collision detection

**Comment by CnL26** [comment]
Thats dope i tried it in pydroid with hackers keyboard and its amazing im
just shocked to see my pixel art used at all! Thank you for getting me into
graphics! I cant wait till its finished. You got real skillz!!!

**Comment by CnL26** [comment]
https://github.com/CnL26/MyPixelArtChars/blob/main/Lorenzo's_Snake_Assets/LnzsApple_Tree.png

**Comment by itspyguru** [comment]
wow it looks cool. You are really getting good at it.

**Comment by itspyguru** [comment]
I was thinking if its possible to have this tree a little animation. This will make it more dynamic. Can you make 3-4 frames for this. I have updated the game with the tree in the classic and the boxed mode.

**Comment by CnL26** [comment]
Its 32x48 pixels. I even included a gif to show the time delay of each frame which is 0.3 seconds...I hope you like it😁

**Comment by CnL26** [comment]
https://github.com/CnL26/MyPixelArtChars/tree/main/Lorenzo's_Snake_Assets/LrnzsAppleTree

**Comment by itspyguru** [comment]
Thankyou very much bro !

**Comment by CnL26** [comment]
I was hoping you'd like it!!!:-)

**Comment by CnL26** [comment]
Hey check this video at the 1:18:27 mark it may help with the snake's imgs:
https://youtu.be/QFvqStqPCRU

**Comment by CnL26** [comment]
If need be, I can rotate the imgs to add extra angles as shown in the vid. Just let me know! Also if you take the four edges, i.e., the dirt; You can put them together to make a block/obstacle I will upload an example on here soon!

**Comment by itspyguru** [comment]
> I was hoping you'd like it!!!:-)

It was beautiful and its now available in the classic and boxed mode.

**Comment by itspyguru** [comment]
> If need be, I can rotate the imgs to add extra angles as shown in the vid. Just let me know! Also if you take the four edges, i.e., the dirt; You can put them together to make a block/obstacle I will upload an example on here soon!

I'll be waiting for it. Till then let me watch the video.

**Comment by itspyguru** [comment]
Sorry for the extra work bro. But there are few mismatch in the snake skins and my game variables. Here's what i am looking for
* 1 head, 1 tail, 1 body and 4 corners in 16x16 pixels
* The skin of some other shade which is visible on green grass. 
* All assets in png. Currently the tail was in gif.

Currently you can play the game and see the snake is not visible. I am not looking for much animations now so don't create too much frames or body parts for the time being.

**Comment by CnL26** [comment]
Ok so what I will do is scale down the body because originally it's at 32px, as well as the skin and change color and I will try and show how it will look put together those are just minor changes and for the tail just use tail00 it's the first frame so it's just a still tail I'm headed to work but as soon as I return home I will have those changes for you👍

**Comment by itspyguru** [comment]
Actually i did tried scaling the assets to 16x16, but then it became so small that its not visible. You can do it whenever you have free time.

**Comment by itspyguru** [comment]
any update ?

**Comment by CnL26** [comment]
Yes I will upload in a hour or so I had to make double check everything

**Comment by CnL26** [comment]
I hope you can use this if so I will upload the rest.

https://github.com/CnL26/MyPixelArtChars/blob/main/Lorenzo's_Snake_Assets/betaSnakeaHead.png

**Comment by CnL26** [comment]
https://github.com/CnL26/MyPixelArtChars/blob/main/Lorenzo's_Snake_Assets/beta_body.png

**Comment by CnL26** [comment]
https://github.com/CnL26/MyPixelArtChars/blob/main/Lorenzo's_Snake_Assets/beta_tail.png

**Comment by CnL26** [comment]
******!!!!! updated!!!!******
Let me know if they are usable
